_posts/2019-04-17-owengamingboy.md

```diff
@@ -0,0 +1,79 @@
+---
+layout:     post
+title:      "Deconstructivist Pop: A Case Study"
+author:     Rahul Tirumalareddy
+date:       2019-04-17
```

Comment:
Shouldn't this be May 25?

Reply:
I started writing the article at that date

Reply:
actually yea, it makes more sense to have the finish date, will update